src/zenml/cli/stack_recipes.py

```diff
@@ -825,15 +871,51 @@ def deploy(
             deployment.
         skip_pull: Skip the pull of the stack recipe before deploying. This
             should be used if you have a local copy of your recipe already.
-        enabled_services: A list of services to install. Choose from mlflow, seldon,
-            kserve, kubeflow, tekton.
+        artifact_store: The flavor of artifact store to deploy. In the case of
+            the artifact store, it doesn't matter what you specify here, as
+            there's only one flavor per cloud provider and that will be deployed.
+        orchestrator: The flavor of orchestrator to use.
+        container_registry: The flavor of container registry to deploy. In the case of
+            the container registry, it doesn't matter what you specify here, as
+            there's only one flavor per cloud provider and that will be deployed.
+        model_deployer: The flavor of model deployer to deploy.
+        experiment_tracker: The flavor of experiment tracker to deploy.
+        secrets_manager: The flavor of secrets manager to deploy. In the case of
+            the secrets manager, it doesn't matter what you specify here, as
+            there's only one flavor per cloud provider and that will be deployed.
+        step_operator: The flavor of step operator to deploy.
+        config: Use a YAML or JSON configuration or configuration file to pass
+            variables to the stack recipe.
+
+    Returns:
+        The exit code of the command. 1 if deployment successful, 0 otherwise.
```

Comment:
the use case was to know if a deployment was cancelled from a prompt and then exit the program without raising an error (since it is not one, technically)/ But I agree that it looks out of place and so I have removed it in favour of using `cli_utils.error()`